code/modules/mob/living/simple_animal/hostile/skeletonjack.dm

```diff
@@ -0,0 +1,32 @@
+/mob/living/simple_animal/hostile/skeletonjack
+	name = "skeleton jack"
+	desc = "This one isn't king of the pumpkin patch."
+	icon = 'icons/obj/candybucket.dmi'
+	icon_state = "skeleton_jack"
+	mob_property_flags = MOB_UNDEAD | MOB_SUPERNATURAL
+	environment_smash_flags = SMASH_LIGHT_STRUCTURES
+	supernatural = 1
+	health = 30	//It's a bunch of bones glued to a pumpkin
+	maxHealth = 30
+	melee_damage_lower = 10
+	melee_damage_upper = 15
+	var/obj/structure/candybucket/candy_jack/ourBucket = null
+
+/mob/living/simple_animal/hostile/skeletonjack/proc/trickOrTreat(var/candyAmount)
```

Comment:
it seems like a confusing naming convention to use the same name for this proc as in the other object but they do entirely different things

Reply:
I didn't even notice I used the same one twice. Man this is an embarrassing amount of changes for something so simple.